.travis.yml

```diff
@@ -1,15 +1,23 @@
 dist: xenial
 language: php
 php:
-  - '7.0'
-  - '7.3'
-  
+- '7.0'
+- '7.3'
 install:
-  - php composer.phar update --no-suggest
-  - php composer.phar install --no-suggest
-  
+- php composer.phar update --no-suggest
+- php composer.phar install --no-suggest
+before_deploy:
+- zip -r release.zip ./*
 before_script:
-  - if find . -name "*.php" ! -path "./vendor/*" -print0 | xargs -0 -n 1 -P 8 php -l | grep -v "No syntax errors detected"; then exit 1; fi
-
+- if find . -name "*.php" ! -path "./vendor/*" -print0 | xargs -0 -n 1 -P 8 php -l
+  | grep -v "No syntax errors detected"; then exit 1; fi
 script:
-#  - ./vendor/squizlabs/php_codesniffer/bin/phpcs --ignore=/vendor/*,/app/Modules/*/Resources/Views/*.php,_ide_helper.php --standard=PSR2 --exclude=PSR2.Files.EndFileNewline,Squiz.Functions.MultiLineFunctionDeclaration,Squiz.WhiteSpace.SuperfluousWhitespace,Squiz.Functions.FunctionDeclaration,PSR2.Methods.FunctionClosingBrace -n --encoding=utf-8 --extensions=php ./*
+- ./vendor/squizlabs/php_codesniffer/bin/phpcs --ignore=/vendor/*,/app/Modules/*/Resources/Views/*.php,_ide_helper.php,/database/*/*.php,/resources/views/*.php,/resources/views/*/*.php --standard=PSR2 --exclude=PSR2.Files.EndFileNewline,Squiz.Functions.MultiLineFunctionDeclaration,Squiz.WhiteSpace.SuperfluousWhitespace,Squiz.Functions.FunctionDeclaration,PSR2.Methods.FunctionClosingBrace -n --encoding=utf-8 --extensions=php ./* 
+deploy:
+  provider: releases
+  api_key:
+    secure: PpIHZtADb9SWSWplTDJyUr7BqlwEJ0xphKsY+qK8y5RVtIQwoyu+UMaldXz+m4C7J7CSA/y4KPjyudo2y8kas6FhybNlSCu5iUwRXw6nupAyfomJfP63tvC6UCN6rAtWPtmH201FhPB+LaWtpRKx4gh/VK46hD0qIMADj1fKY2A=
```

Comment:
I belive you want to remove this from the PR? :smile:

Reply:
Yea, let me rebase this change. The travisci wasn’t needed for this PR